swift/ios-shoppe-demo/Resources/Utilities.swift

```diff
@@ -16,3 +17,9 @@ func getPlist(withName name: String) -> [String: String]? {
 
     return nil
 }
+
+func registerCells(tableView: UITableView, registry: Registry) {
```

Comment:
I think this should be a method on `UITableView` as a custom extension, or a method on `Registry`. Freestanding functions like this are awkward to read.